### ISSUE TITLE
Breaks "Öffi" and "Saarfahrplan" apps

### ISSUE THREAD

**Issue by RalfJung** [microg/android_frameworks_mapsv1#7]
"Öfi" crashes on startup, with
```
E/AndroidRuntime( 3324): java.lang.RuntimeException: Unable to create application de.schildbach.oeffi.Application: java.lang.ClassCastException: org.slf4j.impl.AndroidLoggerFactory cannot be cast to ch.qos.logback.classic.LoggerContext
E/AndroidRuntime( 3324): 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4596)
E/AndroidRuntime( 3324): 	at de.robv.android.xposed.XposedBridge.invokeOriginalMethodNative(Native Method)
E/AndroidRuntime( 3324): 	at de.robv.android.xposed.XposedBridge.handleHookedMethod(XposedBridge.java:668)
E/AndroidRuntime( 3324): 	at android.app.ActivityThread.handleBindApplication(<Xposed>)
E/AndroidRuntime( 3324): 	at android.app.ActivityThread.access$1600(ActivityThread.java:154)
E/AndroidRuntime( 3324): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1382)
E/AndroidRuntime( 3324): 	at android.os.Handler.dispatchMessage(Handler.java:102)
E/AndroidRuntime( 3324): 	at android.os.Looper.loop(Looper.java:135)
E/AndroidRuntime( 3324): 	at android.app.ActivityThread.main(ActivityThread.java:5294)
E/AndroidRuntime( 3324): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime( 3324): 	at java.lang.reflect.Method.invoke(Method.java:372)
E/AndroidRuntime( 3324): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:904)
E/AndroidRuntime( 3324): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:699)
E/AndroidRuntime( 3324): 	at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:115)
E/AndroidRuntime( 3324): Caused by: java.lang.ClassCastException: org.slf4j.impl.AndroidLoggerFactory cannot be cast to ch.qos.logback.classic.LoggerContext
E/AndroidRuntime( 3324): 	at de.schildbach.oeffi.Application.initLogging(Application.java:156)
E/AndroidRuntime( 3324): 	at de.schildbach.oeffi.Application.onCreate(Application.java:44)
E/AndroidRuntime( 3324): 	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1012)
E/AndroidRuntime( 3324): 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4593)
E/AndroidRuntime( 3324): 	... 13 more
```

Saarfahrplan crashes when touching the "map" icon.

**Comment by aphirst** [comment]
As there's no build available for (easy) public testing, is this bug still present?

**Comment by mar-v-in** [comment]
This bug was fixed with upgrading osmdroid to version 5.1.
Regarding builds: Just added one: https://github.com/microg/android_frameworks_mapsv1/releases

**Comment by RalfJung** [comment]
Öffi works fine with that release :-)
